scripts/update_copyright_date.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
```

Comment:
This is the only file that was manually edited.

Reply:
Scripting FTW!!